aws/src/integration/java/org/apache/iceberg/aws/AwsIntegTestUtil.java

```diff
@@ -127,6 +129,47 @@ public static void cleanS3Bucket(S3Client s3, String bucketName, String prefix)
     }
   }
 
+  /**
+   * Method used to clean up a S3 directory bucket which doesn't care about versions
+   *
+   * @param s3 an instance of S3Client to be used to list/delete objects
+   * @param bucketName name of the bucket
+   * @param prefix the path prefix we want to remove
+   */
+  public static void cleanS3DirectoryBucket(S3Client s3, String prefix, String bucketName) {
```

Comment:
looks like bucket name and prefix argument order is wrong? compared to `cleanS3GeneralPurposeBucket(S3Client s3, String bucketName, String prefix)`